platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/GuestClaimsHandlerExt.java

```diff
@@ -108,6 +108,7 @@ public void init() {
             this.profiles = (Map<String, Object>) fromJson(json);
         } catch (IOException e) {
             LOGGER.debug("Could not find profiles.json during installation: ", e);
+            this.profiles = new HashMap<>();
```

Comment:
I suppose either could have worked. At a previous point, `profiles` was allowed to be null. The issue was more-so with `getClaimsProfiles` and not `profiles` being null.